core/src/main/cpp/gdn/gast_node.cpp

```diff
@@ -93,6 +93,7 @@ void GastNode::set_projection_mesh(ProjectionMesh::ProjectionMeshType projection
     }
 
     projection_mesh->set_external_texture(external_texture);
+    update_projection_mesh_camera();
```

Comment:
Is this needed since it's also being updated each frame in `_process`?

Reply:
Removed.